### ISSUE TITLE
 confd seems to have problems loading the config if an instance's NIC has a network

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#421]
Originally reported of Google Code with ID 365.

```
0. create a network and associate it with the current node group
1. create an instance nic with a network
(eg. --net 0:network=name)
2. check confd log:
2013-02-07 14:09:43,609853000000 UTC: ganeti-confd pid=21813 ERROR Failed to load config: parsing configuration: key 'instances': key 'ultrt-gnt0.hot.corp.google.com': key 'nics': key 'network': Unable to read JSObject

network should be and is a string (currently the network name, in the future perhaps the uuid). What is confd expecting there?


```
Originally added on 2013-02-07 14:17:18 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
This was already reported on the list, and Helga pushed a fix for it. Are you sure you have an up-to-date tree?
```
Originally added on 2013-02-07 14:20:41 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
Correct. It is commit 0c6d6a52e by Helga. As I saw, Guido (partially) cherry-picked it to devel-2.7. So I guess it is closed.

Thanks,
dimara
```
Originally added on 2013-02-08 15:23:39 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
Yes, this now works in 2.7

Thanks everybody,

Guido
```
Originally added on 2013-02-08 16:38:39 +0000 UTC.

Changed State: Fixed

**Comment by Ganeti-Issues-Migrator** [comment]
```
Marking 2.7 bugs fixed, as 2.7.0~rc1 was released, and should be ready to be tested in production environments.
```
Originally added on 2013-05-09 05:42:48 +0000 UTC.

Changed State: Released
